<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimizeDialog</class>
 <widget class="QDialog" name="PrimizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Primize Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>840</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_primImg_ImgName">
      <property name="text">
       <string>File Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_primImgName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_primImgSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_primImg_origImg">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>420</width>
     <height>420</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_primImgInputs">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>80</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_primImg_ImgDim">
        <item>
         <property name="text">
          <string>2D Data</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D Data</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4D Data</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_primImg_imgUpl">
        <property name="text">
         <string>Open Image Data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_primImg_run">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
